var/spack/repos/builtin/packages/mrbayes/package.py

```diff
@@ -15,11 +15,14 @@ class Mrbayes(AutotoolsPackage):
     homepage = "http://mrbayes.sourceforge.net"
     git      = "https://github.com/NBISweden/MrBayes.git"
 
-    version('2017-11-22', commit='8a9adb11bcc538cb95d91d57568dff383f924503')
+    version('3.2.7a', commit='0176ac2d0bfed53a5bc0aaf3f5a3def71f23575f')
 
     variant('mpi', default=True, description='Enable MPI parallel support')
     variant('beagle', default=True, description='Enable BEAGLE library for speed benefits')
+    variant('readline', default=True, description='Enable readline library')
     variant('sse', default=True, description='Enable SSE in order to substantially speed up execution')
+    variant('avx', default=True, description='Enable AVX in order to substantially speed up execution')
+    variant('fma', default=True, description='Enable FMA in order to substantially speed up execution')
```

Comment:
@alalazo is there a better way to handle sse/avx/fma now?

Reply:
I wonder whether including these ISA options in `variants` is necessary.  They are all turned on by default and will be automatically turned off during `configure`. Perhaps we can delete the clutter and only include `mpi`, `beagle` and `readline`? For most users it will be more than enough.